tests/kernel/wave/wave_e2e_test.py

```diff
@@ -757,14 +757,14 @@ def test_igemm_conv(n, h, w, c, hf, wf, nf, stride, mem_space, layout, request):
         inputs={
             N: i // SZ_OUT,
             C: j // (HF * WF),
-            H: (i % SZ_OUT) % W_OUT * stride + (j % (HF * WF)) % WF,
-            W: (i % SZ_OUT) // W_OUT * stride + (j % (HF * WF)) // WF,
+            W: (i % SZ_OUT) % W_OUT * stride + (j % (HF * WF)) % WF,

```

Comment:
was this a bug? how were the tests passing if we had H and W switched?

Reply:
This changes how we unroll conv window (rows vs colums first), it should produce semantically equivalent result as long as it synchronized between all mapping. But it also affects memory access pattern and this way it's slightly faster.